### PR TITLE
Multiversion Tutorial Refactor (Use CronJob)

### PR DESCRIPTION
**Pull request by DirectXMan12**
This switches the multiversion tutorial over to continue the work from the CronJob tutorial.  It also refactors things a bit to avoid pages that are too long, and converts some diagrams to SVG, so that they natively support theming.

### CODE REVIEW

File: docs/book/src/multiversion-tutorial/deployment.md [hunk 1]
```diff
@@ -1,73 +1,78 @@
...
+# Deployment and Testing
+
+Before we can test out our conversion, we'll need to enable them conversion in our CRD:
+
+Kubebuilder generates Kubernetes manifests under the `config` directory with webhook
+bits disabled.  To enable them, we need to:
+
+- Enable `patches/webhook_in_<kind>.yaml` and
+  `patches/cainjection_in_<kind>.yaml` in
+  `config/crds/kustomization.yaml` file.
```
Comment (droot) [inline docs/book/src/multiversion-tutorial/deployment.md]: s/crds/crd

File: docs/book/src/multiversion-tutorial/deployment.md [hunk 2]
```diff
@@ -1,73 +1,78 @@
...
 
-Ensure that you have installed [cert-manager](../cronjob-tutorial/cert-manager.md) `0.9.0+` version in your cluster. We have
-tested the instructions in this tutorial with [0.9.0-alpha.0](https://github.com/jetstack/cert-manager/releases/tag/v0.9.0-alpha.0) release.
-
-Running `make deploy` will deploy the controller-manager in the cluster.
+You'll need [cert-manager](../cronjob-tutorial/cert-manager.md) installed
+(version `0.9.0+`) unless you've got some other certificate management
+solution.  The Kubebuilder team has tested the instructions in this tutorial
+with
+[0.9.0-alpha.0](https://github.com/jetstack/cert-manager/releases/tag/v0.9.0-alpha.0)
```
Comment (droot) [inline docs/book/src/multiversion-tutorial/deployment.md]: 0.9.0 is available now.

**Comment by mengqiy** [comment]
@DirectXMan12 I guess the comments have to be resolved in a followup PR now

**Pull request merged**